lib/PEX.ts

```diff
@@ -95,7 +95,7 @@ export class PEX {
    */
   public evaluateCredentials(
     presentationDefinition: IPresentationDefinition,
-    verifiableCredentials: OriginalVerifiableCredential[],
+    verifiableCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[],
```

Comment:
This is literally what OriginalVerifiableCredential is

Reply:
fixed